platforms/ios/src/TangramMap/TGMapViewController.h

```diff
@@ -530,6 +532,32 @@ NS_ASSUME_NONNULL_BEGIN
 - (int)loadSceneFileAsync:(NSString *)path sceneUpdates:(NSArray<TGSceneUpdate *> *)sceneUpdates;
 
 /**
+ Loads a scene synchronously using an explicitly specified yaml string, with a list of updates
+ to be applied to be applied to the scene.
+ If a scene update error happens, scene updates won't be applied.
+
+ @param yaml Yaml scene string
+ @param resourceRoot Base path to resolve relative urls within the yaml scene string
+ @param sceneUpdates a list of `TGSceneUpdate` to apply to the scene
+
+ @return the integer (SceneID) associated with this scene load or -1 if scene can not be loaded.
+*/
+- (int)loadSceneYaml:(NSString *)yaml resourceRoot:(NSString *)resourceRoot sceneUpdates:(NSArray<TGSceneUpdate *> *)sceneUpdates;
```

Comment:
We could have a convenience method defined as such:
```
- (int)loadSceneFromYaml:(NSString *)yaml updates:(NSArray<TGSceneUpdate *> *)updates;
```
where `resourceRoot` defaults to the `Resource` application folder (`[[NSBundle mainBundle] resourceURL]`).